tgui/packages/tgfont/static/tgfont.css

```diff
@@ -1,7 +1,7 @@
 @font-face {
     font-family: "tgfont";
-    src: url("./tgfont.woff2?45c3c7acc69dd413375d77898d24e41e") format("woff2"),
```

Comment:
What are you doing with the fonts?

Reply:
Its the only way to implement the icon. It is the method outlined by Anturk.

Reply:
when new icons are added, rebuild of fonts and css is required, which changes the hash of the font file